ml-agents/mlagents/trainers/learn.py

```diff
@@ -73,6 +78,12 @@ def run_training(sub_id: int, run_seed: int, run_options, process_queue):
             docker_target_name=docker_target_name
         )
 
+    sampler = None
+    lesson_config = None
+    if sampler_file_path is not None:
```

Comment:
Should raise an explicit error if `(sampler_file_path and not lesson_config_path)` (or may sampler_file_path xor lesson_config_path)